crossterm_style/src/lib.rs

```diff
@@ -1,6 +1,19 @@
 //! A module that contains all the actions related to the styling of the terminal.
 //! Like applying attributes to text and changing the foreground and background.
 
+use std::fmt::Display;
+
+pub use crossterm_utils::{execute, queue, Command, ExecutableCommand, QueueableCommand, Result};
+
+use self::ansi_color::AnsiColor;
+pub use self::color::{color, PrintStyledFont, SetAttr, SetBg, SetFg, TerminalColor};
+pub use self::enums::{Attribute, Color, Colored};
+pub use self::objectstyle::ObjectStyle;
+pub use self::styledobject::StyledObject;
+pub use self::traits::{Colorize, Styler};
+#[cfg(windows)]
+use self::winapi_color::WinApiColor;
```

Comment:
This probably isn't supposed to be here. I see that it already was there, but wondering why....

Reply:
```rust
pub use self::styledobject::StyledObject;
```

What about `pub use self::objectstyle::ObjectStyle`? Same category, no? You'd like to remove these two reexports, because they're accessible via public `objectstyle` & `styledobject` modules? If yes, examples needs to be updated as well, because they do use crate root reexport.

```rust
pub use self::traits::{Colorize, Styler};
```

These traits are used in examples and this reexports them, because the `traits` module isn't public (`mod traits`).

```rust
#[cfg(windows)]
use self::winapi_color::WinApiColor;
```

Seems useless, no usage in the `lib.rs`. Wondering why Travis CI log doesn't say unused import. Will remove it.

Reply:
It think that `WinApiColor` should be imported directly. Now it is re-exported at the top level of the crate so that all files have it when doing `use super::*`

Reply:
`WinApiColor` import already moved the `src/color.rs` where it's used.